_data/faq.yml

```diff
@@ -458,46 +458,59 @@
 
-                    The database is stored in an internal folder on your device (not accessible without root access). Typically this database will only have a size of some megabyte.
-                    For the database you can choose in the c:geo settings whether it is stored internally or on the SD card.
-
-                2. The additional data (e.g. images, maps) for all the caches stored in the database
-
-                    This additional data can be much larger as it contains the images of the cache description and (optionally) the images retrieved from logs and (optionally) static maps.
-                    The additional data is stored on SD card by default (in a folder called ".cgeo").
+                    You may store the database of c:geo either system internally or in a directory on your user storage.
+                    
+                    You can change the database location in Menu → Settings → System → "Database location". \\
+                    As the database availability is critical for using c:geo and the typical size is only some megabyte, we decided not to allow moving it to the external SD card.
+                    If you enable the option "On user storage" the database will be moved to the folder /Android/data/cgeo.geocaching/files/databases (may vary depending on your device).
 
-                Some users are complaining, that the additional data as well as the database (if external storage is selected) are still stored in the internal memory of their devices.
-                We always place the folder on the storage medium that the device provides as SD card. On some devices, this storage medium is part of the internal memory, which is mounted as an SD card (It's like an SD card fixed mounted in your device).
+                2. The additional data (e.g. images, maps) for all the caches stored in the database:
 
-                Unfortunately there is no standardized method to find the location of a real SD card on your device. There are some workarounds but those might fail on some devices, thus we decided not to implement them for now.
-                Furthermore starting with Android 4.4 write access to the real SD card is forbidden for apps.
+                    This additional data can be much larger as it contains the images of the cache description and (optionally) the images retrieved from logs and (optionally) static maps. \\
+                    By default this data is stored on the internal user storage in /Android/data/cgeo.geocaching/files/GeocacheData/.
+                    
+                    You can move this data to your external SD card in Menu → Settings → System → "Geocache Data Directory". \\
+                    If your device supports using the external SD card this menu option will show you the available target directories as well as the current size of your geocache data and the available space on the target directories.
```

Comment:
Should we mention the restriction on API Level 19?
Change is only possible if the target dir has enough free space. Maybe this is obvious.